### DISCUSSION TITLE
5.4 update broken?

### DISCUSSION THREAD

**Discussion by gerroon** [ampache/ampache#3282] [Q&A]
```
Ampache Update
This page handles all database updates to Ampache starting with 3.3.3.5. Your current version is 5.4.0-release with database version 530016

The following updates need to be performed:

Version: Version: 5.4 Build.: Build:0000
* Index `title` with `enabled` on `song` table to speed up searching
Version: Version: 5.4 Build.: Build:0001
* Index `album` table columns.
* `catalog`, `album_artist`, `original_year`, `release_type`, `release_status`, `mbid`, `mbid_group`
Version: Version: 5.4 Build.: Build:0002
* Index `object_type` with `date` in `object_count` table
```

I press Update Now,

```
Ampache Update
This page handles all database updates to Ampache starting with 3.3.3.5. Your current version is 4.4.2-release with database version 530016

The following updates need to be performed:

No Update Needed [Return to main page]
```

This again sends me back to the Update page after landing on `/public`

**Comment by lachlan-00** [comment]
i don't see anything obvious in the update page, it looks like it should have completed ok.

You can't just click on `Return to main page` or go back to the index/login pages?

**Comment by gerroon** [comment]
When I click on the return page, it sent me back to the update database page.     This is same as me trying to open the login page, it always sent me back to the update page. I went back to the previous version to be able to use Ampache for now.


I did this process of updating the databse in the app many times, not sure why it does not work now.  I will try it again later.

The only thing I can see is that my folder content is different than the zip I downloaded

![ampache](https://user-images.githubusercontent.com/8519469/169749456-566f4077-0cb4-4bdc-9bdd-4906dd22b03e.png)
